change-notes/support/README.md

```diff
@@ -0,0 +1,6 @@
+# Files moved to ``docs`` directory
+
+Now that all the QL documentation is in the `Semmle/ql` repository,
```

Comment:
```suggestion
Now that all of the QL documentation is in this repository,
```
Suggesting this change, because to me using the full name of the repo you are already in sounds a little bit strange.